portfolio/src/main/webapp/script.js

```diff
@@ -190,4 +192,13 @@ function deleteComments(commentId) {
         const maxComments = document.getElementById('maxcomments').value;
         getComments(maxComments)
```

Comment:
JavaScript is somewhat unique among the popular languages in that you can omit positional arguments from function calls, like is happening here for the new `filterInput` argument. I suggest triggering this scenario and watching the Network tab in your browser console to see what happens. (If you haven't used it yet, here's a shortish video about it: https://www.youtube.com/watch?v=nl8iKlo2NeM)

Reply:
Ooo, I didn't know that and had just missed changing that line before. I'll make some modifications now that I know this is possible in javascript, thanks!